package.json

```diff
@@ -1005,8 +1005,7 @@
           "id": "confluent-flink-artifacts",
           "name": "Flink Artifacts",
           "visibility": "collapsed",
-          "icon": "$(confluent-logo)",
-          "when": "false"
+          "icon": "$(confluent-logo)"
```

Comment:
icons, icons

Reply:
I wouldn't change this icon since it's only for the view when in a sidebar that's toggled between "label" and "icon" mode - custom icons are down around here: https://github.com/confluentinc/vscode/blob/6b73761ae8f2204b2f3fbe0e4eeed1684b7b96da/package.json#L1689-L1702

Reply:
oh nice thank you!